### DISCUSSION TITLE
v7.2.1

### DISCUSSION THREAD

**Discussion by Ghost-chu** [PBH-BTN/PeerBanHelper#811] [公告]
v7.2.1 是 v7.2.0 的一个首日补丁，用于修复 v7.2.0 中已知的错误。

## 错误修复

* 修复 .deb 安装包安装时可能因为文件夹已经存在导致报错的问题 @Anuskuss
* 修复警报提示框中警报内容文本丢失的问题 @Gaojianli 
* 修复启动时 IPFS Share BanList 提示出错，导致 0 号测试组用户无法启动 PBH 的问题，我们暂时禁用了此功能，并将随下一个修复版本一同修复。 @Ghost-chu

## Docker

DockerHub: `ghostchu/peerbanhelper:v7.2.0`  
阿里云国内镜像加速: `registry.cn-hangzhou.aliyuncs.com/ghostchu/peerbanhelper:v7.2.0`

v7.2.0 更新日志请见：https://github.com/PBH-BTN/PeerBanHelper/releases/tag/v7.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/PBH-BTN/PeerBanHelper/releases/tag/v7.2.1'>v7.2.1</a>.</em>

**Comment by happilying** [comment]
docker7.2.0是个什么鬼🤣
